src/Experimental/isomorphism.jl

```diff
@@ -173,9 +171,40 @@ function has_isomorph(
     vertex_relation::Union{Nothing,Function}=nothing,
     edge_relation::Union{Nothing,Function}=nothing,
 )::Bool
-    return has_isomorph(
-        g1, g2, alg; vertex_relation=vertex_relation, edge_relation=edge_relation
-    )
+    throw(MethodError(has_isomorph, (g1, g2, alg)))
+end
+
+"""
+    canonize!(g, alg::IsomorphismAlgorithm=AlgNautyGraphs())
+
+Permute the vertices of graph `g` into the canonical order defined by the algorithm `alg` and return the permutation. 
+If graphs `g1` and `g2` are isomorphic, the orders of their vertices will be equal after canonizing them with the same algorithm.
+
+### Optional Arguments
+- `alg`: The algorithm that is used to canonize the graph. Can be only be `AlgNautyGraphs()`
+   at this moment, which requires `NautyGraphs` to be installed and imported.
+
+### Examples
+```doctest.jl
+julia> canonize!(path_graph(3))
+[1, 3, 2]
+
+julia> g1 = path_digraph(4)
+julia> g2 = path_digraph(4)[[2, 3, 1, 4]]
+julia> g1 == g2
+false
+julia> canonize!(g1)
+[4, 2, 3, 1]
+julia> canonize!(g2)
+[4, 1, 2, 3]
+julia> g1 == g2
+true
+```
+### See also 
+[`has_isomorph`](@ref)
+"""
+function canonize!(g::AbstractGraph, alg::IsomorphismAlgorithm=AlgNautyGraphs())
+    throw(MethodError(canonize!, (g, alg)))
 end
```

Comment:
this can be `function canonize! end` so that we do not manually throw `MethodError`. Then we can have a hint that detects the method error and warns that one needs to import NautyGraphs.